### DISCUSSION TITLE
Problem with the 3.5mm mini-jack: loose connection

### DISCUSSION THREAD

**Discussion by stardusthamster** [cardonabits/haxo-hw#89] [Q&A] [answered]
As time goes by I get the following problem using the haxophone: The audio connection is sometimes broken, it is an on-off problem. I checked the mini-jack connection and found out that it moves if you push on it even if it is soldered on and the soldering points look ok and there nothings moves. So when I push it slightly the sound is fine again and then gets lost after a while.

What I wanted to ask:
Is there another audio-jack I can use that is more robust? Or is there another possibility to fix this? 
Any ideas?

Thank you very much!

**Comment by jcard0na** [comment, marked as answer]
Hi @stardusthamster,

This is the audio-jack we ended up using in the campaign build:  https://jlcpcb.com/partdetail/boomele_boom_Precision_elec-PJ320B/C18594

If it is damaged, it should be easy to swap.  These other parts seem to be footprint compatible:

https://jlcpcb.com/partdetail/XkbConnection-PJ_320BSMT/C2884998
https://jlcpcb.com/partdetail/Korean_HropartsElec-PJ_320B3A/C707224
https://jlcpcb.com/partdetail/Sofng-PJ320B/C22355831

Let me know if you end up swapping the connector, and if that improves things for you.
Cheers,

**Reply by stardusthamster** [reply]
Thank you very much for the details. It took me some time to order the spare part.
I got it in a nice letter from italy:
![haxo_jack_1](https://github.com/user-attachments/assets/07a387cf-881a-443a-a77e-8e4a95f4ccbf)

After I soldered it on, it works fine again!

**Reply by jcard0na** [reply]
Thanks for letting me know!  Just for future reference, which one of the three suggested alternative parts did you end up choosing?

**Reply by stardusthamster** [reply]
I tried to order from the links you mentioned, but as I understood, it is only possible to order there when also ordering a PCB.

So I had to look elsewhere and found one on e-bay that is footprint compatible, I searched for PJ 320 and found this one 
https://www.ebay.de/itm/266421441362

![pj320](https://github.com/user-attachments/assets/b6b02570-c89f-4606-8a95-766ec7a13005)

It fits perfectly!

So I don't really know which version of PJ 230 it really is...

**Comment by jcard0na** [comment]
Oh, I should have mentioned that... all JLCPCB part numbers can be purchased from LCSC.  Just enter the JLCPCB part number on LCSC and you get the part. Here you have the links, just in case anyone has the same problem:

https://www.lcsc.com/product-detail/Audio-Connectors-Headphones_XKB-Connection-PJ-320B-SMT_C2884998.html
https://www.lcsc.com/product-detail/Audio-Connectors-Headphones_Korean-Hroparts-Elec-PJ-320B-3A_C707224.html
https://www.lcsc.com/product-detail/Audio-Connectors-Headphones_SOFNG-PJ-320B_C22355831.html
